cmd/server/cadence/cadence.go

```diff
@@ -135,13 +139,86 @@ func BuildCLI() *cli.App {
 		{
 			Name:    "start",
 			Aliases: []string{""},
-			Usage:   "start cadence notification service",
+			Flags: []cli.Flag{
+				cli.StringFlag{
+					Name:  "services",
+					Value: "receiver, notifier",
+					Usage: "set up webhook test endpoint",
+				},
+			},
+			Usage: "start cadence notification service",
 			Action: func(c *cli.Context) {
-				startHandler(c)
+				var wg sync.WaitGroup
+				services := getServices(c)
+
+				for _, service := range services {
+					wg.Add(1)
+					go launchService(service, c)
+				}
+
+				wg.Wait()
 			},
 		},
 	}
-
 	return app
+}
+
+func launchService(service string, c *cli.Context) {
+	switch service {
+	case "notifier":
+		startHandler(c)
+		break
+	case "receiver":
+		startTestWebhookEndpoint()
+		break
+	default:
+		log.Printf("Invalid service: %v", service)
+	}
+}
+
+func getServices(c *cli.Context) []string {
+	val := strings.TrimSpace(c.String("services"))
+	tokens := strings.Split(val, ",")
+
+	if len(tokens) == 0 {
+		return []string{"notifier"}
```

Comment:
We have set default value for this parameter. So I think it will make more sense to error on this case so that it’s not surprising.